src/test/java/org/openrewrite/github/ActionsSetupJavaAdoptOpenj9ToSemeruTest.java

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Only one test because it is the same `SetupJavaDistributionReplacerVisitor` that is well tested in the other recipe